### DISCUSSION TITLE
nil on  local tl = require("tl")

### DISCUSSION THREAD

**Discussion by yuriv** [teal-language/tl#476] [Q&A]
I am a newbie in Lua. After installing Teal (luarocks install tl) I am tried to load tl in luajit console and have got the result:

LuaJIT 2.0.4 -- Copyright (C) 2005-2015 Mike Pall. http://luajit.org/
JIT: ON CMOV SSE2 SSE3 SSE4.1 fold cse dce fwd dse narrow loop abc sink fuse
> local tl = require("tl")
> print(tl)
nil

On regular Lua a have got same result:

Lua 5.1.4  Copyright (C) 1994-2008 Lua.org, PUC-Rio
> local tl = require("tl")
> print(tl)
nil
> 

Why? I am on CentOS 7.9.

Thanks in advance

**Comment by catwell** [comment]
Are you running those commands in the REPL?

You cannot use local variables like this in the REPL. Remove `local` from your example or wrap it in a do/end block:

```
Lua 5.1.5  Copyright (C) 1994-2012 Lua.org, PUC-Rio
> local tl = require("tl")
> print(tl)
nil
> do
>> local tl = require("tl")
>> print(tl)
>> end
table: 0x561663403be0
```

**Reply by yuriv** [reply]
I works, thank you.

**Reply by yuriv** [reply]
It works, sorry
